src/BizHawk.Emulation.Common/Interfaces/Services/ISaveRam.cs

```diff
@@ -28,5 +28,10 @@ public interface ISaveRam : IEmulatorService
 		/// This value should be considered a hint more than an absolute truth.
 		/// </summary>
 		bool SaveRamModified { get; }
+
+		/// <summary>
+		/// Certain cores may support SaveRam only in certain situations, for example only for certain games.
+		/// </summary>
+		bool SupportsSaveRam { get; }

```

Comment:
Cores can conditionally implement services by mutating the service provider from in the ctor body; can that be done for `ISaveRam`?

Reply:
I think so. I don't know how much work it would be to make the necessary modifications.